lib/datahub-client/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+- Return domain metadata for Charts
```

Comment:
```suggestion
- Return domain metadata for Charts
- Added cache
```

Reply:
Those changes are in the main django app, but this changelog is just for the library bit (since we are still publishing it separately)